Dockerfile

```diff
@@ -1,6 +1,4 @@
 ARG GO_IMAGE=rancher/hardened-build-base:v1.20.7b3
-ARG TAG="v1.8.10"
-ARG ARCH="amd64"
 FROM ${GO_IMAGE} as base-builder
```

Comment:
Can you also remove the `"` from the TAG defined on this file so we don't have an empty PR that only removes the `"`